src/controller/BookController.ts

```diff
@@ -0,0 +1,114 @@
+import { bookRepository as repository } from '../database/repositories';
+import Book from '../entity/Book';
+
+export default {
+
+    async create(ctx: any) {
+
+        const {
+            uuid,
+            name,
+            isbn,
+            author,
+            releaseDate
+        } = JSON.parse(JSON.stringify(ctx.request.body));
+    
+        const book = repository.create({
+            uuid: uuid,
+            name: name,
+            isbn: isbn,
+            author: author,
+            releaseDate: releaseDate
+        });
+    
+        ctx.body = await repository.save(book);
+    },
+
+    async getAll(ctx: any) {
+        let books: Book[];
+        
+        if (ctx.request.url.includes('take') && ctx.request.url.includes('skip')) {
+            const take = ctx.query['take'];
+            const skip = ctx.query['skip'];
+            books = await repository.find({
+                take,
+                skip
+            });
+        }
+        else if (ctx.request.url.includes('sort')) {
```

Comment:
Also great work with a sorting!

Can't use sort and pagination together, better to remove `else` so one can both add skip/take + sort to the query.
just
```ts
if (ctx.request.url.includes('sort')) {
```